src/Stratis.Bitcoin.Features.PoA/BasePoAFeatureConsensusRules/HeaderTimeChecksPoARule.cs

```diff
@@ -13,7 +13,7 @@ namespace Stratis.Bitcoin.Features.PoA.BasePoAFeatureConsensusRules
     public class HeaderTimeChecksPoARule : HeaderValidationConsensusRule
     {
         /// <summary>Up to how many seconds headers's timestamp can be in the future to be considered valid.</summary>
-        public const int MaxFutureDriftSeconds = 60;
+        public const int MaxFutureDriftSeconds = 5;
```

Comment:
Why value of 5? 

Such a low value reduces cost of sibyl attack with invalid time provided in the handshake and all it takes to make miner produce invalid blocks is to change it's perception of time by just 5 seconds. 

5 will work but it's better to have targetSpacing - 1

Reply:
If we do TargetSpacing - 1, then someone could be 15 seconds ahead and still have their blocks accepted, and they would block anyone who is more than 1 second behind. But we should always want to accept blocks 1 second behind.

So then the value that makes sense is 8. You can be +-8 seconds to the clock and noone screws anyone else.

But then... Apparently we need to allow 3 seconds for a kind of "lag" that can exist between nodes due to latency and processing time, so we do 8 - 3 = 5.

This is what I understand, mostly from @maciejzaleski 's analysis

Reply:
>Such a low value reduces cost of sibyl attack with invalid time provided in the handshake

Can you explain this a little more ?